src/main/java/core/basesyntax/service/ReportCreatorImpl.java

```diff
@@ -0,0 +1,24 @@
+package core.basesyntax.service;
+
+import java.util.Map;
+
+public class ReportCreatorImpl implements ReportCreator {
+    private static final String COLUMN_NAMES = "fruit,quantity";
+    private static final String SEPARATOR = ",";

```

Comment:
this constant is duplicated, make it public in the class that is related to it the most and reuse existing one in other classes